integral_view/core/storage_access/views/cifs_share_management.py

```diff
@@ -514,13 +514,23 @@ def update_samba_server_settings(request):
                     raise Exception(err)
                 if cd["security"] == "ads":
                     rc, err = cifs.kinit(
-                        "administrator", cd["password"], cd["realm"])
+                        cd["username"], cd["password"], cd["realm"])
                     if err:
-                        raise Exception(err)
+                        if "Password incorrect" in err:
+                            raise Exception("Invalid Credentials, Please try again")
+                        elif "not found" in err:
+                            raise Exception("Could not find an Active Directory user with the username: '%s'! " % cd["username"])
+                        elif cd["password"] in err:
```

Comment:
Because the original error message looked a bit cryptic and couple of users complained about the `echo password | kinit` statement in the error message. So we decided add these exception messages.